### DISCUSSION TITLE
Uploaded Documents and Intrustry Session Progresses seems to be erased !

### DISCUSSION THREAD

**Discussion by Akhil-Tony** [github/india#240] [Students]
hello friends , 
excited to see you all ,
today unfortunately when logging into my account it seem all my data has been erased ,anybody facing similar problems ???????????

**Comment by VivekJain-01** [comment]
same

**Reply by Aditya-Bhate** [reply]
same

**Comment by Akhil-Tony** [comment]
now alright , may be a temporary technical glitch

**Reply by VivekJain-01** [reply]
nope not the same case with me.

**Reply by Mayank-6387992885** [reply]
it cannot be uploaded ... after clicking submit ...

**Reply by AyushSenDazzler** [reply]
In addition all my progress is also removed, i have completed all sessions but now it reflecting incomplete

**Comment by Mik-27** [comment]
Refer #261 
It worked for me.

**Comment by Kamaruddheen** [comment]
> hello friends , excited to see you all , today unfortunately when logging into my account it seem all my data has been erased ,anybody facing similar problems ???????????

Yeah man, looks like they purposely disabled all the features in **Google Chrome** except other browsers (Firefox, Edge) I guess so. I tried on Firefox and Edge browser. It is working fine.

**Comment by TheSuranaverse** [comment]
**FUTURE READY SEPARATE CHANNEL**

For smooth communication & discussion!
Link: https://discord.gg/ceMXzhfaka

Thank you 🙏
